src/components/CommentsArea/style.jsx

```diff
@@ -60,6 +60,63 @@ const SCommentsArea = styled.section`
       justify-content: flex-end;
     }
   }
+
+  @media only screen and (max-width: 700px) {
+    flex-direction: column;
+    align-items: center;
+    margin: auto;
+
+    .commentSection {
+      width: 90%;
+      margin: auto;
+    }
```

Comment:
Exactly the same as the precedent media query, making it unnecessary code duplication, should be removed